### DISCUSSION TITLE
Download `shfmt` from GitHub releases

### DISCUSSION THREAD

**Discussion by latipun7** [mason-org/mason.nvim#468] [API suggestions]
Currently [`shfmt`](https://github.com/mvdan/sh) is installed using `go`. I'm not `go` developer so installing `go` which is so huge imo only for installing `shfmt` is kinda overkill.

[`shfmt` GitHub release](https://github.com/mvdan/sh/releases) already has the latest version of pre-build of that tools, but it's not zip or tarball. They are the executable itself. So, after downloading that, it just needs to be renamed into `shfmt` or `shfmt.exe` and set the executable.

Does this approach could be supported? Thank you very much for this great tool.

**Comment by williamboman** [comment]
Hello! Yes, definitely. Prebuilt packages are generally preferred when available. PR to change this is very much welcomed, https://github.com/williamboman/mason.nvim/pull/434 may be used as a reference (it transformed an installer from using go to GitHub release assets). Otherwise I'll get around to it eventually!

**Reply by latipun7** [reply]
Thanks. I'll create new issue from this discussion for tracking.
